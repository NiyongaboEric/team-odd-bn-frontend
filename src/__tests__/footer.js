import React from 'react';
import { shallow } from 'enzyme';
import Footer from '../components/footer';

describe('Footer', () => {
  const wrapper = shallow(<Footer />);

  it('should render without crash', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
