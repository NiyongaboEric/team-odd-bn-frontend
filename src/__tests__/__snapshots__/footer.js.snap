// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer should render without crash 1`] = `ShallowWrapper {}`;
